frontend/packages/console-app/src/components/access-modes/access-mode.tsx

```diff
@@ -33,6 +33,7 @@ export const AccessModeSelector: React.FC<AccessModeSelectorProps> = (props) =>
     availableAccessModes = [],
     description,
     ignoreReadOnly,
+    spAccessMode,
```

Comment:
```suggestion
    accessMode,
```
we just want to give this component an initial value, not a storage profile metadata, can we call the initial value `accessMode`

Reply:
accessMode is already taken by the state

Reply:
changed to `accMode`